pkg/testutil/alert_validation_test.go

```diff
@@ -116,4 +116,136 @@ var _ = Describe("Validators", func() {
 			Expect(problems[0].Description).To(ContainSubstring("alert must have a description annotation"))
 		})
 	})
+
+	Context("Custom alert Validations", func() {
```

Comment:
I think a different file for these unit tests would make sense

`pkg/testutil/alert_custom_validation_test.go`

Reply:
Done